test/e2e/service.go

```diff
@@ -797,6 +797,75 @@ var _ = Describe("Services", func() {
 	})
 })
 
+var _ = Describe("Services", func() {
+	f := NewFramework("services-temp")
+
+	var t *explorerTest
+	BeforeEach(func() {
+		t = newDefaultExplorerTest(f.Client, f.Namespace.Name)
+	})
+
+	It("should serve round-robin services round-robin", func() {
```

Comment:
Doesn't this actually test that the pods behind the service just got the same number of requests?  Rather than that they were served in strict round-robin order?  If so, best have the text reflect that more accurately.